examples/slice/slice_utils.py

```diff
@@ -0,0 +1,341 @@
+import firedrake as fd
```

Comment:
Can we move this file to a slice directory in `utils` like the shallow water utils?